ci/build-gmt.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# Build and install GMT
+
+# To return a failure if any commands inside fail
+set -e
+
+cat > cmake/ConfigUser.cmake << 'EOF'
+set (CMAKE_INSTALL_PREFIX "$ENV{INSTALLDIR}")
+set (GMT_LIBDIR "$ENV{INSTALLDIR}/lib")
+set (DCW_ROOT "$ENV{COASTLINEDIR}")
+set (GSHHG_ROOT "$ENV{COASTLINEDIR}")
+
+set (CMAKE_BUILD_TYPE Debug)
+enable_testing()
+set (DO_EXAMPLES TRUE)
+set (DO_TESTS TRUE)
+set (DO_ANIMATIONS TRUE)
+set (N_TEST_JOBS 2)
+EOF
+
+if [ "$COVERAGE" == "true" ]; then
+    cat >> cmake/ConfigUser.cmake << 'EOF'
+set (CMAKE_C_FLAGS "-Wall -Wdeclaration-after-statement -coverage -O0")
+EOF
+fi
```

Comment:
We don't need to enable the tests in compile-only jobs, so we can change it to:
```
if [ "$TEST" == "all"]; then
    cat >> cmake/ConfigUser.cmake << 'EOF'
set (CMAKE_BUILD_TYPE Debug)
enable_testing()
set (DO_EXAMPLES TRUE)
set (DO_TESTS TRUE)
set (DO_ANIMATIONS TRUE)
set (N_TEST_JOBS 2)
set (CMAKE_C_FLAGS "-Wall -Wdeclaration-after-statement -coverage -O0")
EOF
fi   
```

Reply:
Are any of those required for building the docs?

Reply:
Never mind, already found it.